x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/process/autodetect/AutodetectProcessManager.java

```diff
@@ -333,10 +330,16 @@ public void writeUpdateProcessMessage(JobTask jobTask, UpdateParams updateParams
                     updateProcessMessage.setFilter(filter);
 
                     if (updateParams.isUpdateScheduledEvents()) {
-                        jobManager.getJob(jobTask.getJobId(), new ActionListener<Job>() {
+                        jobManager.getJob(jobTask.getJobId(), new ActionListener<>() {
                             @Override
                             public void onResponse(Job job) {
-                                DataCounts dataCounts = getStatistics(jobTask).get().v1();
```

Comment:
Intellij flagged this as a get on an optional without a isPresent check which is a very valid warning. In practice I don't think we could hit an NPE here but it shows the value in the inspections _if you can keep the noise level down_.